partitura/io/exportmusicxml.py

```diff
@@ -634,12 +638,20 @@ def merge_measure_contents(notes, other, measure_start):
         merged[0] = []
         cost[0] = 0
 
+    # CHANGE: disabled cost-based merging of non-note elements into stream
+    # because this led to attributes not being in the beginning of the measure,
+    # which in turn led to problems with musescore
+    # fix: add atributes first, then the notes.
+    # problem: unclear whether this cost-based merging will break anything or
+    # was just cosmetic to avoid too many forwards and backwards.
+    # related issue: https://github.com/CPJKU/partitura/issues/390
+
     # get the voice for which merging notes and other has lowest cost
-    merge_voice = sorted(cost.items(), key=itemgetter(1))[0][0]
+    # merge_voice = sorted(cost.items(), key=itemgetter(1))[0][0]
```

Comment:
I'm not sure if this might end up having some unintended effect down the line... In any case, I think this change is ok as long as it doesn't break anything on the tests. Would it be possible to have a documented list of scores/pieces that had an issue with the sorting (i.e., the piece(s) that prompted this issue)? This could be useful for further testing down the line